recipe/meta.yaml

```diff
@@ -44,13 +44,15 @@ requirements:
     - python-dateutil >=2.5
     - backports_abc  # [py34]
     - pint >0.8
-    - numba 
+    - numba
+    - numexpr
     - hyperspy-gui-ipywidgets
+    - sparse
 test:
     requires:
         - pytest
         - pytest-mpl
-        - freetype 2.8*
+        - freetype 2.9.*
```

Comment:
freetype >= 2.8 instead of pinning it?

Reply:
It's a requirement of mpl 2.2.3. I've tested it locally and it's fine.